1_data_prep/12_render_instance_ids.py

```diff
@@ -0,0 +1,97 @@
+import sys
+from pathlib import Path
+
+import geopandas as gpd
+import pandas as pd
+
+# Library imports
+from geograypher.cameras.derived_cameras import MetashapeCameraSet
+from geograypher.entrypoints.render_labels import render_labels
+from geograypher.meshes import TexturedPhotogrammetryMesh
+from geograypher.utils.visualization import show_segmentation_labels
+
+import _bootstrap
+from configs.path_config import path_config
+
+# The image is downsampled to this fraction for accelerated rendering
+RENDER_IMAGE_SCALE = 0.1
+# Portions of the mesh within this distance of the labels are used for rendering
+MESH_BUFFER_RADIUS_METER = 20
+# Cameras within this radius of the annotations are used for training
+CAMERAS_BUFFER_RADIUS_METERS = 25
+# Downsample target
+DOWNSAMPLE_TARGET = 1
+
+# CRS inferred from automate-metashape config that generated the product, metadata.xml, or sibling rasters
+INPUT_CRS = "EPSG:26910"
+
+# Render data from this column in the geofile to each image
+# "unique_ID" column is the ID assigned to each tree-crown by the GeometricTreeCrownDetector
+LABEL_COLUMN_NAME = "unique_ID"
+VIS = True
+
+RENDER_ROOT_FOLDER = path_config.rendered_instance_ids / "renders"
+VIS_ROOT_FOLDER = path_config.rendered_instance_ids / "visualizations"
```

Comment:
It might be worth creating a new variable for this so someone doesn't need to recreate this logic in a later script.